api/src/main/java/org/openmrs/module/eptsreports/reporting/calculation/pvls/RoutineCalculation.java

```diff
@@ -107,7 +111,7 @@ public CalculationResultMap evaluate(Collection<Integer> cohort, Map<String, Obj
 				artInitiationDate = (Date) artStartDateResult.getValue();
 			}
 			// check that this patient should be on ART for more than six months
-			if (artInitiationDate != null && lastVlObs != null && lastVlObs.getObsDatetime() != null) {
+			if (artInitiationDate != null && lastVlObs != null && lastVlObs.getObsDatetime() != null && criteria != null && onArtForMoreThan3Months.contains(pId)) {
```

Comment:
@ningosi why are we adding this condition here when it wasn't here before?

Reply:
I was following what was written as TO DO in the java doc
/**
	 * Patients on ART for the last X months with one VL result registered in the 12 month period
	 * Between Y to Z months after ART initiation TODO: merge with
	 * RoutineForBreastfeedingAndPregnantWomenCalculation definition
	 * 
	 * @param cohort
	 * @param params
	 * @param context
	 * @return CalculationResultMap

Reply:
Should I revert?

Reply:
Sorry, I should have been more specific. I am referring to adding the `onArtForMoreThan3Months` condition